chminfo_service/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2015, wootski (https://github.com/wootski).
```

Comment:
I agree with you. Copyright needs to be enforceable with an actual name or backing organization.